src/analysis/normed/group/pointwise.lean

```diff
@@ -24,6 +25,7 @@ variables {E : Type*}
 section seminormed_group
 variables [seminormed_group E] {ε δ : ℝ} {s t : set E} {x y : E}
 
+-- note: we can't use `lipschitz_with.bounded_image2` here without adding `[isometric_smul E E]`
 @[to_additive] lemma metric.bounded.mul (hs : bounded s) (ht : bounded t) : bounded (s * t) :=
```

Comment:
@YaelDillies, any ideas on whether I can make a better generalization that does work here?

Reply:
Yes, I believe you can instead use
```
lemma bounded_image2 (f : α → β → γ) {K₁ K₂ : ℝ≥0} {s : set α} {t : set β}
  (hf₁ : ∀ b ∈ t, lipschitz_with K₁ (λ a, f a b) s) (hf₂ : ∀ a ∈ s, lipschitz_on_with K₂ (f a) t)
  (hs : bounded s) (ht : bounded t) : bounded (set.image2 f s t) := sorry
```
Then you use the fact that multiplication is Lipschitz on bounded sets.

Reply:
I generalized `bounded_image2`, but haven't changed this proof

> Then you use the fact that multiplication is Lipschitz on bounded sets.

Where is this result?

Reply:
Not in mathlib :/

Should be easy though. The bound on the set turns into the Lipschitz constant.

Reply:
Are you sure this doesn't need the same `[isometric_smul E E]` argument that I was trying to avoid?

Reply:
I don't think so? What it needs is `dist (c • x) (c • y) = ‖c‖ * dist x y`, which is different from `isometric_smul E E` (`dist (c • x) (c • y) = dist x y`.

Reply:
Don't forget we're in weird `normed_group` land not `normed_add_group`.

Reply:
Then I'm confused: Isn't `isometric_smul E E` always true in that context?

Reply:
Not without commutativity; only `isometric_smul (MulOpposite E) E` is true

Reply:
Do you care about noncommutative settings?

Reply:
> Do you care about noncommutative settings?

No, but I don't want to un-generalize a lemma solely for the point of a golf.